shared/src/main/scala/coop/rchain/shared/Stopwatch.scala

```diff
@@ -17,6 +17,14 @@ object Stopwatch {
       _  <- log(s"$tag [${showTime(m)}]")
     } yield a
 
+  def durationRaw[F[_]: Sync, A](block: => F[A]): F[(A, FiniteDuration)] =
```

Comment:
This function is almost the same as `duration` except mapping second element of a tuple with `showTime`.
`durationRaw(block).map(_.map(showTime))`